spec/support/content_schema_helpers.rb

```diff
@@ -1,20 +1,20 @@
-module Mocha
-  module ParameterMatchers
-    def valid_payload_for(format_name)
-      ValidSchemaMatcher.new(format_name)
-    end
+# module Mocha
```

Comment:
Why did this need to be disabled and why only temporarily?

Reply:
It was copied across, but was blocking any test running. I've actually removed it entirely now, it's not really possible to do the test in the same way, and it didn't really add much.